__tests__/news/news.mutation.test.js

```diff
@@ -60,10 +66,15 @@ const existingNews = {
 };
 
 let newsId = '';
+let token;
 const newsDoesNotExistId = '1f2ad410eb01783384e6111b';
+const invalidToken =  'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOiI1ZjQ3NTMxYjVjNWYzMTJkMDhlYjdkZDUiLCJlbWFpbCI6ImFkbWluQGdtYWlsLmNvbSIsImlhdCI6MTYwMDA2NTE1AaaaAAaaAaaaAaAaAAA4AaAaaA.A9-aAAaaaaaa-D5ghd96IjsKq_r18tD97WhzI7QXfbA';
 
-describe('test news mutations', () => {
-  test('#1 should add news to database', async () => {
+describe('Create news test', () => {
```

Comment:
Please, put all of the `describe` statements for news into one singe `describe` with name `News mutations tests`

Reply:
Changed!